mlflow/store/model_registry/databricks_workspace_model_registry_rest_store.py

```diff
@@ -0,0 +1,31 @@
+from mlflow.store.model_registry.rest_store import RestStore
+from mlflow.exceptions import MlflowException
+
+
+def _raise_unsupported_method(method, message=None):
+    messages = [
+        f"Method '{method}' is unsupported for models in the Workspace Model Registry. "
+        f"Upgrade to Models in Unity Catalog to access the latest features. You can configure "
+        f"the MLflow Python client to access models in Unity Catalog by running "
+        f"mlflow.set_registry_uri('databricks-uc') before accessing models.",
+    ]
+    if message is not None:
+        messages.append(message)
+    messages.append("See the user guide for more information")
```

Comment:
```suggestion
    messages.append("See the user guide for more information.")
```

Reply:
btw what is the user guide?

Reply:
Good question, we had that in from private preview - maybe it's better to say "See Databricks docs for more information". The most ideal would be to include a cloud-specific link to Databricks docs, but it's also a bit risky in case doc links are moved around/change over time (although typically we should add a redirect in our docs in that case)

Reply:
Added the "." and switched to "See Databricks docs" wording

Reply:
Actually, let me just remove "See Databricks docs" since it's a bit redundant / meaningless, for example we don't have that after all error messages in OSS/Databricks